packages/dm-core-plugins/src/view_selector/Content.tsx

```diff
@@ -28,6 +28,7 @@ export const Content = (props: {
         <HidableWrapper
           key={config.viewId}
           hidden={config.viewId !== selectedView}
+          role="tabpanel"
```

Comment:
Nice 👍🏻 but on a sidenote - is it a bit weird that we're using css to set display: none instead of just not rendering the content the usual react way? I mean either way is fine, but we pretty much always do the latter